<!-- launch example -->

<launch>
	<group ns="fmProcessors">
		<node pkg="frobit_lidar" name="obstacle_detect" type="obstacle_detect_node.py" output="screen">
			<param name="scan_sub" value="/fmInformation/laser_scan"/>
			<param name="obstacle_pub" value="/fmKnowledge/obstacle"/>
			<param name="ahead_threshold_warning" value="2.50"/> 
			<param name="lateral_threshold_warning" value="0.5"/> 
			<param name="ahead_threshold_alarm" value="1.5"/> 
			<param name="lateral_threshold_alarm" value="0.5"/> 
			<param name="angular_resolution" value="1"/> 
			<param name="minimum_range" value="0.05"/> 
			<param name="scans_skip" value="1"/> 
		</node>
	</group>
</launch>

